cmd/yggctl/actions.go

```diff
@@ -402,10 +414,13 @@ func generateWorkerDataAction(ctx *cli.Context) error {
 		if err != nil {
 			return cli.Exit(fmt.Errorf("cannot create file %v: %v", d.FilePath, err), 1)
 		}
-		defer f.Close()
 		if err := d.Template.Execute(f, config); err != nil {
+			_ = f.Close()
```

Comment:
May be still worth catching.

Reply:
No, we are already terminating the application, because it was not possible to write to file. Serious error can happen, when data were cached and data are written during `f.Close()`, but this IO error was already cough at line 417 and it is processing at this point.

Reply:
Processing the error would only allow us to print an additional message that the file close failed.